third-party/build.sh

```diff
@@ -135,23 +187,28 @@ cd grpc
 git submodule update --init
 
 #For parallel compilation, make -j is selected. For systems with limited DRAM, avoid using -j option for all make commands
-make -j 
+#make -j 
+mkdir -p cmake/build
+cd cmake/build
+cmake ../.. -DBUILD_SHARED_LIBS=ON -DgRPC_INSTALL=ON -DCMAKE_INSTALL_PREFIX=$ABS_BUILD_DIR
+#cmake ../.. -DgRPC_INSTALL=ON -DCMAKE_INSTALL_PREFIX=$ABS_BUILD_DIR
+make -j
```

Comment:
make -j will fail on a system with lower memory. And the errors appear as internal compiler error and do not give enough hints to analyze build failure. This problem is often seen with nvmm build.
Should we add a check to obtain memory size? If DRAM size is greater than some threshold(say 32GB), then issue a make -j. Else use make.